### ISSUE TITLE
sign in from app - something is incomplete

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#6861]
If you sign in to cocalc -- at least in cocalc-docker -- via https://localhost:9000/settings 

<img width="1621" alt="image" src="https://github.com/sagemathinc/cocalc/assets/1276278/0b4d8c8e-7232-464d-9208-b12dcbd8589b">

then for me it appears to work, but seems somehow incomplete, like some cookie isn't set.

If you instead sign in via https://localhost:9000/auth/sign-in

<img width="1178" alt="image" src="https://github.com/sagemathinc/cocalc/assets/1276278/37f81e20-585a-4b64-96d7-87b5c5aa139a">

then it works fine, though the cookie doesn't last very long if you are using an invalid security cert, but that is I think a browser security constraint.

This suggests a cookie isn't being properly set when signing in via https://localhost:9000/settings, but it is set properly via  https://localhost:9000/auth/sign-in

The point of this issue is either to fix this, or to just redirect  https://localhost:9000/settings to  https://localhost:9000/auth/sign-in.

I would prefer to fix this to avoid interrupting workflows when re-authenticating, but it's also complicated supporting two separate sign in implementations.

**Comment by williamstein** [comment]
It might be there is no difference, and just browsers now have VERY severe limits on https cookies when using SSL that is not properly configured.  Maybe the fix for "safe environments" is that we add a different less safe mode that uses non-https cookies for auth?

**Comment by williamstein** [comment]
OK, I just got to the bottom of this.  I'm running two different cocalc servers on localhost on different ports.  The auth cookie for one gets set just fine... overwriting the other one, and conversely, back and forth.  Every time I sign into to one server, it breaks sign in for the other. The solution is to put the port in the name of the auth cookie, which is not being done.

Right now the base_url is in the remember_me cookie name, but the port isn't. That's a problem.

I have added support for a new env variable COCALC_REMEMBER_ME_COOKIE_NAME.  If you set it then that is the name of the cookie that will be used.  So just set it differently for each server you start.
